### DISCUSSION TITLE
Why is my output js file name a hash?

### DISCUSSION THREAD

**Discussion by srishti-s** [wojtekmaj/react-pdf#983] [Q&A]
I've started using this library, and while everything works fine, I noticed in my dist that the name of the js that's copied over is a has. How  can I rename this to be just pdf-worker.js or something better:

<img width="200" alt="image" src="https://user-images.githubusercontent.com/54010712/163495709-232d7622-1b5f-41e4-8f35-c371eab80dc4.png">

**Comment by wojtekmaj** [comment]
Which entry file are you using?
